src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -386,6 +386,10 @@ def create_premis_object(fileUUID):
     for elem in create_premis_object_characteristics_extensions(fileUUID):
         objectCharacteristics.append(elem)
 
+    creatingApplication = etree.Element(ns.premisBNS + "creatingApplication")
+    etree.SubElement(creatingApplication, ns.premisBNS + "dateCreatedByApplication").text = str(f.modificationtime)[:10]
```

Comment:
I would feel more comfortable using a public function like `strftime("%Y-%m-%d")` where the output format is explicit. You're relying on a `__str__` method of `datetime.datetime`, perhaps that's not as reliable?

`f.modificationtime.isoformat()` may also be okay? Date and time combined seem like a good fit here, isn't it?